source/extensions/filters/http/rate_limit_quota/client_impl.cc

```diff
@@ -0,0 +1,63 @@
+#include "source/extensions/filters/http/rate_limit_quota/client_impl.h"
+
+#include "source/common/tracing/http_tracer_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace RateLimitQuota {
+
+void RateLimitClientImpl::onRemoteClose(Grpc::Status::GrpcStatus, const std::string&) {
+  // TODO(tyxia) Add implementation later.
+  stream_closed_ = true;
```

Comment:
This will need to be wired into the client, so that stream closures can be retried

Reply:
Do you mean `stream_closed_`? It is currently in the client. Could you elaborate on this?